display_api/display.lua

```diff
@@ -251,6 +251,15 @@ function display_api.on_destruct(pos)
 	end
 end
 
+function display_api.on_blast(pos, intensity)
+	if not minetest.is_protected(pos, "tnt:blast") then
```

Comment:
from the docs
`name will be "" for non-players or unknown players.`
probably shouldnt be passing a node name style string in